### PR TITLE
Update glusterfs example according to config best practices

### PR DESCRIPTION
**Pull request by linzichang**
related to #11361

### CODE REVIEW

**Comment by bgrant0607** [comment]
Awaiting resolution of #14256

**Comment by bgrant0607** [comment]
@k8s-bot unit test this

File: examples/glusterfs/glusterfs-pod.json [hunk 1]
```diff
@@ -1,32 +0,0 @@
-{
-    "apiVersion": "v1",
-    "id": "glusterfs",
```
Comment (saad-ali) [inline examples/glusterfs/glusterfs-pod.json]: Was `id` removed on purpose from the yaml version?

**Comment by saad-ali** [comment]
LGTM mod nit

**Comment by saad-ali** [comment]
@linzichang do you plan update this PR?

**Comment by linzichang** [comment]
No, recently I have no time to work on this.

**Comment by saad-ali** [comment]
Ok, I'll close this PR. If anyone else wants to pick it up they can, create a new PR.

**Pull request closed without merging**